src/graphic/Fast3D/gfx_pc.h

```diff
@@ -118,9 +119,22 @@ struct RawTexMetadata {
 };
 
 struct ShaderMod {
-    bool enabled = false;
-    int16_t id;
-    uint8_t type;
+    const char* vertex;
+    const char* fragment;
+
+    std::optional<std::string> GetVertex() const {
```

Comment:
Why use `std::optional` here, when we can use empty string to mean "no shader is set"?